internal/server/data/organization.go

```diff
@@ -1,11 +1,32 @@
 package data
 
 import (
+	"context"
+
 	"gorm.io/gorm"
 
 	"github.com/infrahq/infra/internal/server/models"
 )
 
+type orgCtxKey struct{}
+
+// OrgFromContext returns the Organization stored using WithOrg.
+func OrgFromContext(ctx context.Context) *models.Organization {
+	org, ok := ctx.Value(orgCtxKey{}).(*models.Organization)
+	if !ok {
+		// TODO(orgs): panic("no org")
```

Comment:
I think we should merge this with the panic or create an issue to track adding the panic, this feels dangerous otherwise

Reply:
Adding the panic is what causes tons of tests to break and makes the change difficult. I think an issue would be good. 

I created https://github.com/infrahq/infra/issues/2903 to track these kinds of things.